teps/0025-hermekton.md

```diff
@@ -55,16 +56,14 @@ I would like to add this support to Tekton Pipelines, and use this feature (and
 
 ### Goals
 
-* Allow Task authors to indicate that their Task can/should be run hermetically
-* Allow TaskRun authors to designate particular runs to run hermetically
-* Allow Pipeline authors to designate that parts or all of a pipeline can/should be run hermetically
-* Allow PipelineRun authors to designate particular runs to run hermetically
-* Allow post-build auditing to show clearly which *Runs were run hermetically
+* Allow Task authors to indicate Steps that can/should be run hermetically
+* Allow TaskRun authors to indicate Steps that can/should be run hermetically
+* Allow post-build auditing to show clearly which Steps were run hermetically
```

Comment:
im wondering about whether we want to replace the previous list of goals, or add these new ones onto an existing list?

i think there might be cases where you'd want to specify this at the level of the entire task (esp if we make it easier to combine tasks together)